packages/scope-manager/src/referencer/Reference.ts

```diff
@@ -1,13 +1,14 @@
 import type { TSESTree } from '@typescript-eslint/types';
 
-import { createIdGenerator } from '../ID';
 import type { Scope } from '../scope';
 import type { Variable } from '../variable';
 
+import { createIdGenerator } from '../ID';
+
 enum ReferenceFlag {
   Read = 0x1,
-  Write = 0x2,
   ReadWrite = 0x3,
+  Write = 0x2,
 }
```

Comment:
Value vs label ordering for enums again.

Perhaps we should disable sorting on enums?